FTPizza.Tests/FTP.Test.cs

```diff
@@ -1,11 +1,28 @@
 ﻿using NUnit.Framework;
 using System.Collections.Generic;
+using System.IO;
+using System;
 
 namespace FTPizza.Tests
 {
     [TestFixture]
     public class FTPTests
     {
+        public string[] connectionFile = File.ReadAllLines(@"connection.txt");
+        public string ftpUrl;
+        public string ftpUser;
+        public string ftpPassword;
+        public Ftp client;
+
+        [SetUp]
+        public void Init()
+        {
+            ftpUrl = connectionFile[0].Replace("url=", "");
+            ftpUser = connectionFile[1].Replace("user=", "");
+            ftpPassword = connectionFile[2].Replace("pass=", "");
+            client = new Ftp(ftpUser, ftpPassword, ftpUrl);
+        }
+
```

Comment:
Matt and I did discuss this yesterday, and agree it isn't the most elegant or correct way to perform a unit test.. However we figured a quick and dirty approach that would make it possible for the rest of the team to write unit tests was preferable given the short amount of time we have left before the assignment is due.

If you have a suggestion to approach this differently, by all means share it. 

We can also put this to a vote...